app/controllers/api/v1/schedules_controller.rb

```diff
@@ -0,0 +1,52 @@
+module Api
+  module V1
+    class SchedulesController < ExternalApiController
+      # This Endpoint is not using JSON API format!
+      def create
+        return forbidden unless current_user.admin?
+
+        store = Store.find_by(id: params['store-id'])
```

Comment:
Why not `Store.find`? =)

Reply:
@simaob 
AR's `.find` throws an exception when it cannot find anything.